test/tools/WebListener/Controllers/GetController.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Http.Extensions;
+using mvc.Models;
+
+namespace mvc.Controllers
+{
+    public class GetController : Controller
+    {
+        public JsonResult Index()
+        {
+            Hashtable args = new Hashtable();
+            foreach (var key in Request.Query.Keys)
+            {
+                args.Add(key, String.Join(" ,", Request.Query[key]));
+            }
+            Hashtable headers = new Hashtable();
+            foreach (var key in Request.Headers.Keys)
+            {
+                headers.Add(key, String.Join(" ,", Request.Headers[key]));
```

Comment:
We use " ," twice - it is good to use a const.
Do we really have space?

Reply:
I will create a constant, as for the space, it should be the other way around `", "` and yes, the space is there when multiple values are supplied for a given header.

```
X-Fake-Header: Value1
X-Fake-Header: Value2
```

and 

```
X-Fake-Header: Value1, Value2
```

Are equivalent.